DAL/user.DAL.js

```diff
@@ -0,0 +1,42 @@
+// get all employees -- need to move ot DAL
+getAllUsers = () => {
+    return {1 : "Gilad", 2 : "Adva", 3: "Ofir", 4: "Dave"};
```

Comment:
`{1 : "Gilad", 2 : "Adva", 3: "Ofir", 4: "Dave"}`
the data should be imported from outside
👆do not mix layers